src/components/Navbar.js

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import '../assets/css/navbar.css';
+import logo from '../assets/images/logo.svg';
 
 const Navbar = () => (
   <div>
-    <h1>BOOKSTORE APP</h1>
-    <nav>
-      <ul>
-        <li>
-          <Link className="link" to="/Books">
-            Books
+    <header className="m-b bg-white round-top">
+      <div className="center max-width-90 flex-row">
+        <h1 className="app-title">BOOKSTORE CMS APP</h1>
+        <div className="cat-title">
+          <Link className="text-grey" to="/BookList">
+            Book
           </Link>
-        </li>
-        <li>
-          <Link className="link" to="/Categories">
+        </div>
+        <div className="cat-title">
+          <Link className="text-grey" to="/Categories">
             Categories
           </Link>
-        </li>
-      </ul>
-    </nav>
+        </div>
+        <img className="react-logo" src={logo} alt="logo" />
+      </div>
+    </header>
   </div>
```

Comment:
- [ ] It will be nice to ensure your nav matches that of the design(eg icon)
- Your nav
![Screenshot from 2021-10-01 16-41-51](https://user-images.githubusercontent.com/65476715/135651825-4c12dce3-9ab5-499f-971b-4f4d86f7ccf5.png)
- Required design
![Screenshot from 2021-10-01 17-00-40](https://user-images.githubusercontent.com/65476715/135651824-2f073474-8de9-4189-a0ca-12607595d40f.png)
.